content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,90 @@
+:menuselection:`Settings --> Users & Companies --> Groups`.
 
-.. image:: access_rights/groups-menus.png
+.. image:: access_rights/click-users-and-companies.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab menus in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Groups.
 
-*Access Rights* rules are the first level of rights. The field is composed of the object name, which
-is the technical name given to a model. For each model, enable the following options as appropriate:
+To create a new group, click :guilabel:`NEW`, select an application, and complete the New Group form
+(detailed below). To modify existing groups, click on a group and edit the contents of the form.
 
-- *Read*: the values of that object can be only seen by the user.
-- *Write*: the values of that object can be edited by the user.
-- *Create*: values for that object can be created by the user.
-- *Delete*: the values of that object can be deleted by the user.
+.. important::
+   Always test the settings being changed to ensure they are being applied to the correct users.
+
+The group form contains multiple tabs for managing all elements of the group. In each tab, click
+:guilabel:`Add a line` to add a new row for users or rules, and click the :guilabel:`X` icon to
+remove a row.
 
-.. image:: access_rights/groups-access-rights.png
+.. image:: access_rights/groups-form.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab access rights in Odoo
+   :alt: Click each tab in the groups form to modify the settings of the group.
 
-| As a second layer of editing and visibility rules, *Record Rules* can be formed. They overwrite,
-  or refine, the *Access Rights*.
-| A record rule is written using a *Domain*. Domains are conditions used to filter or searching
-  data. Therefore, a domain expression is a list of conditions. For each rule, choose among the
-  following options: *Read*, *Write*, *Create* and *Delete* values.
+- **Users Tab**: Lists the current users in the group, users listed in black have administrative
+  rights.
+- **Inherited Tab**: Inherited means that users added to this group are automatically added to the
+  ones listed on this tab. For example: If the group *Sales/Administrator* lists the group
+  *Website/Restricted Editor* in its Inherited tab, then any users added to the
+  *Sales/Administrator* group automatically receive access to the *Website/Restricted Editor* group
+  as well.
+- **Menus Tab**: Defines which menus/models the group can have access to.
+- **Access Rights Tab**: Lists the first level of rights (models) that this group has access rights
+  to. In this tab, the *Model* column represents the common name of the menu/model, and the *Name*
+  column represents the technical name given to the model. For each model, enable the following
+  options as appropriate:
```

Comment:
What can I find this information to make changes? Ie in dev mode hovering over a field in the model